packages/daimo-common/src/model.ts

```diff
@@ -1,9 +1,9 @@
+import { ForeignToken } from "@daimo/contract/src/foreignToken";
```

Comment:
nit: see some imports like this, should be via the package

Reply:
probably also a lint rule that can be added in future

Reply:
fixed, except in `scratchpad`